mlir/docs/MIOpen/tensor_layout.md

```diff
@@ -0,0 +1,89 @@
+# Layout with group
+<br/>
+## tensorflow
+### NHWC
+<br/>
+<br/>
+#### When you want to add group in tensorflow,
+#### you split on dim of c or k , and tf.concat outputs 
+<br/>
+##### axis=3 split on c dim ,input layout become [N H W G C]
+input_groups = tf.split(axis=3, num_or_size_splits=groups, value=x)
+this function will split [N, H, W, C] tensor x along the 3rd dimension, result in groups number of tensors 
+with [N, H, W, C/groups] each ,so it's equivalent to [N H W G C]
```

Comment:
What does `value=x` and `tensor x` means here?

Reply:
it's tf api ,add description,
tensor x is a tensor..... , we use it as an example